search-app/resources/templates/search-eosdis-directory-links.html

```diff
@@ -3,23 +3,38 @@
 {% block title %}{{ app-title }} - Directory of Landing Pages for EOSDIS Collections{% endblock %}
 
 {% block main-content %}
-<h2 class="cover-heading">Directory of Collections Landing Pages</h2>
+<h2 class="cover-heading">Provider Holdings Directory</h2>
 <h3 class="cover-heading">EOSDIS</h3>
 
 <p>
-  Listed here are all the EOSDIS providers linked to pages that list the
-  collection landing pages. Note that the number of collections for each
-  provider is given in parentheses after the provider ID.
+  Listed below are all the EOSDIS providers along with their
```

Comment:
Should this read "Listed below are all _of_ the EOSDIS providers..."? Just read a little funky upon first read.